lib/tasks/manage.rake

```diff
@@ -66,6 +66,20 @@ namespace :manage do
     puts "#{subscribers.count} active subscribers moved from #{from_slug} to #{to_slug}"
   end
 
+  def find_subscriber_list_by_title(title:)
+    subscriber_lists = SubscriberList.where("title LIKE '%#{title}%'")
```

Comment:
Do we want this to be case sensitive? If not, we could use `ILIKE` instead of `LIKE`.

Reply:
Good point thanks! Have updated this now 😄